parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -476,7 +493,10 @@ static Context dataContext(Map<String, String> config) {
         int bloomFilterMaxBytes =
             PropertyUtil.propertyAsInt(
                 config, PARQUET_BLOOM_FILTER_MAX_BYTES, PARQUET_BLOOM_FILTER_MAX_BYTES_DEFAULT);
-        Preconditions.checkArgument(bloomFilterMaxBytes > 0, "bloom Filter Max Bytes must be > 0");
```

Comment:
Is this change relevant? If not, can we do it in a follow-up PR? We need to fix the docs as well.

Reply:
This change is not relevant. I saw the case is not correct, so fixed it `bloom Filter` -> `Bloom filter`